### PR TITLE
Add JenkinsFiles for Windows

### PR DESCRIPTION
**Pull request by AdamBrousseau**
- Add Windows variables
- Add to nightly build
- Add PullRequest Builds

[skip ci]

Issue #1662 
~Depends #1803~
~Currently Blocked by https://github.com/eclipse/openj9/issues/1662#issuecomment-385605381~
~Depends ibmruntimes/openj9-openjdk-jdk8#83~

### CODE REVIEW

**Comment by pshipton** [comment]
Note the ip-validation failure

**Comment by AdamBrousseau** [comment]
jenkins compile aix jdk9

File: buildenv/jenkins/variables/defaults.yml [hunk 1]
```diff
@@ -128,3 +128,29 @@ aix_ppc-64_cmprssptrs:
...
+    10: '/cygdrive/c/openjdk/jdk9'
+  release:
+    8: 'windows-x86_64-normal-server-release'
+    9: 'windows-x86_64-normal-server-release'
+    10: 'windows-x86_64-normal-server-release'
+  freemarker: '/cygdrive/c/openjdk/freemarker.jar'
+  extra_configure_options:
+    8: '--with-freetype-include=/cygdrive/c/openjdk/freetype-2.5.3/include --with-freetype-lib=/cygdrive/c/openjdk/freetype-2.5.3/lib64 --disable-ccache'
+    9: '--with-freetype-src=/cygdrive/c/openjdk/freetype-2.5.3 --with-toolchain-version=2013 --disable-ccache'
+    10: '--with-freetype-src=/cygdrive/c/openjdk/freetype-2.5.3 --with-toolchain-version=2013 --disable-ccache'
```
Comment (DanHeidinga) [inline buildenv/jenkins/variables/defaults.yml]: why `--disable-ccache`?

Comment (AdamBrousseau) [reply buildenv/jenkins/variables/defaults.yml]: We don't have ccache installed on Windows. Adopt also has this option for their Windows builds.

**Comment by DanHeidinga** [comment]
@AdamBrousseau Is this ready to be reviewed & merged?

**Comment by AdamBrousseau** [comment]
No. Still working through issues getting 8 to compile
https://github.com/eclipse/openj9/issues/1662#issuecomment-386495873

**Comment by AdamBrousseau** [comment]
Excluded the Windows 8 Pipeline
https://ci.eclipse.org/openj9/view/Pipelines/job/Pipeline-Build-Test-JDK8-win_x86-64_cmprssptrs/
See https://openj9.slack.com/archives/C8QU16TDW/p152595271600019
I think this can be merged while we continue to debug 8

**Comment by AdamBrousseau** [comment]
Note we still only have 1 machine configured though.
@sxa555 Have you had a chance to setup the remaining 3 Windows boxes?

**Pull request merged**